File: 268-missing-number/missing-number.cpp
class Solution {
public:
    int missingNumber(vector<int>& nums) {
        int res=0;
        int n=nums.size();
        for(int i=0;i<n;i++){
            res=res^nums[i];
            
        }
        for(int i=0;i<=n;i++){
            res=res^i;
        }
        return res;
    }
};

File: 268-missing-number/missing-number.java
class Solution {
    public int missingNumber(int[] nums) {
        int n=nums.length;
        int osum=n*(n+1)/2;
        int sum=0;
        for(int i=0;i<n;i++){
            sum=sum+nums[i];
        }
        int mvalue=osum-sum;
        return mvalue;
    }
}